packages/smooth_app/lib/generic_lib/widgets/smooth_product_image.dart

```diff
@@ -60,6 +60,18 @@ class SmoothProductImage extends StatelessWidget {
                               progress.expectedTotalBytes!,
                         ),
                       ),
+            errorBuilder:
+                (BuildContext context, Object error, StackTrace? stackTrace) =>
+                    SmoothProductImageContainer(
```

Comment:
Looks strange to see a `SmoothProductImageContainer` inside another `SmoothProductImageContainer`. Wouldn't it be enough to put just `Center(SvgPicture.asset(...)` in the `errorBuilder`?

Reply:
Yeah that works , thanks for the suggestion